multiaccounts/accounts/database.go

```diff
@@ -864,3 +868,55 @@ func (db *Database) ENSName() (string, error) {
 	}
 	return "", err
 }
+
+func (db *Database) SaveCommunitySettings(communitySettings params.CommunitySettings) error {
```

Comment:
In protocol/communities/persistence.go live all the functions related to communities CRUD operations. What do you think of having these community settings functions live there too?

Reply:
Happy to move it over!

Main reason I put them here was because I consider them more "settings"  related to the account. Like.. we could also call the entire thing "node_settings" if we want (once we get to actually supporting all the settings like throttling etc). In that sense it wouldn't necessarily be community specific anymore.

But yea, I totally see where you're coming from, so I'll move it over for now. We can still migrate it to somewhere else once this stuff gets broader responsibility.